### ISSUE TITLE
Move temporary demo code in primary.py to demo_primary.py

### ISSUE THREAD

**Issue by vladimir-v-diaz** [uptane/obsolete-reference-implementation#61]
While working on the demo+attack doc, I came across code in the reference implementation that prints banners, which should instead exist in the demo modules.  After speaking with @awwad, he stated that these banner statements are temporary, and will eventually be moved to the demo modules (e.g., demo.demo_primary.py instead of uptane/clients/primary.py).  Opening this ticket so that it's documented in GitHub, instead of a comment in a module.

See:  https://github.com/uptane/uptane/blob/79a5dae1a4f91ec6ba59143163064838485e68ed/uptane/clients/primary.py#L458-L463

**Comment by awwad** [comment]
Yeah, the reference implementation would otherwise keep running along, processing updates for other ECUs. Thanks for creating the issue.

There are two obvious ways to get the information about this back to the demo, where a banner can be printed:
- Make these updates occur in per-Secondary calls to the Primary reference implementation from the demo Primary, instead of one call for all ECUs, so that return values can report the result of the update attempt.
- Have the Primary reference implementation return at the end a report of statuses for each ECU so that the demo Primary can produce banners or other notifications.

I guess I lean toward the second, which is easier. The first may make things cleaner but may also make the workflow harder to grok.
